language/cloud-client/v1/set_endpoint_test.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import set_endpoint
+
+
+def test_set_endpoint(capsys):
+    set_endpoint.set_endpoint()
+
+    out, _ = capsys.readouterr()
+    assert 'Sentiment' in out
```

Comment:
in this case the test does not really give a way of confirming that the endpoint has been set correctly, right?

Reply:
Technically no, but if it doesn't throw an error then it is working.